index.bs

```diff
@@ -537,6 +544,62 @@ partial interface HTMLIFrameElement {
     </div>
   </section>
 </section>
+
+<section>
+  <h2 id="reporting">Reporting</h2>
+  <p><dfn>Feature policy violation reports</dfn> indicate that some behavior of
+  the <a>Document</a> has <a>violated</a> a feature policy.</p>
+
+  <p><a>Feature policy violation reports</a> have the <a>report type</a>
+  "feature-policy-violation".</p>
+
+  <p><a>Feature policy violation reports</a> are <a>visible to
+  <code>ReportingObserver</code>s</a>.
+
+  <pre class="idl">
+    interface FeaturePolicyViolationReportBody : ReportBody {
+      readonly attribute DOMString featureId;
+      readonly attribute DOMString? sourceFile;
+      readonly attribute long? lineNumber;
+      readonly attribute long? columnNumber;
+      readonly attribute DOMString disposition;
+    };
+  </pre>
+
+  A <a>feature policy violation report</a>'s [=report/body=], represented in
+  JavaScript by {{FeaturePolicyViolationReportBody}}, contains the following
+  fields:
+
+    - <dfn for="FeaturePolicyViolationReportBody">featureId</dfn>: The string
+      identifying the <a>policy-controlled feature</a> whose policy has been
+      <a>violated</a>. This string can be used for grouping and counting related
+      reports.
+
+    - <dfn for="FeaturePolicyViolationReportBody">message</dfn>: A
+      human-readable string with details typically matching what would be
+      displayed on the developer console. The message is not guaranteed to be
+      unique for a given [=FeaturePolicyViolationReportBody/featureId=]
+      (e.g. it may contain additional context on how the API was used).
+
+    - <dfn for="FeaturePolicyViolationReportBody">sourceFile</dfn>: If known,
+      the file where the <a>violation</a> occured, or null otherwise.
+
+    - <dfn for="FeaturePolicyViolationReportBody">lineNumber</dfn>: If known,
+      the line number in [=FeaturePolicyViolationReportBody/sourceFile=] where
+      the <a>violation</a> occured, or null otherwise.
+
+    - <dfn for="FeaturePolicyViolationReportBody">columnNumber</dfn>: If known,
+      the column number in [=FeaturePolicyViolationReportBody/sourceFile=] where
+      the <a>violation</a> occured, or null otherwise.
+
+    - <dfn for="FeaturePolicyViolationReportBody">disposition</dfn>: A string
+      indicating whether the <a>violated</a> feature policy was enforced in this
+      case. [=FeaturePolicyViolationReportBody/disposition=] will be set to
+      "enforce" if the policy was enforced, or "report" if the <a>violation</a>
+      resulted only in this report being generated (with no further action taken
+      by the user agent in response to the violation).
```

Comment:
Can we add a note here that says that there is currently no mechanism for enabling a report-only mode, so all reports generated will have this field set to "enforce" ? Otherwise, people may go looking for that, to try to figure out how to enable it.